src/three-components/PMREMGenerator.ts

```diff
@@ -27,7 +29,9 @@ import {encodings, getDirectionChunk, getFaceChunk, texelIO} from './shader-chun
 export const generatePMREM =
```

Comment:
This approach is becoming sophisticated enough that some additional detail would be really helpful here for future readers.

Reply:
💯